inc/namespace.php

```diff
@@ -56,6 +56,13 @@ function should_block_publish() {
  * Enqueue browser assets for the editor.
  */
 function enqueue_assets() {
+	global $pagenow;
+
+	// Ensure this is an edit post page.
+	if ( empty( $pagenow ) || ! in_array( $pagenow, [ 'post.php', 'post-new.php' ], true ) ) {
```

Comment:
What if you're using the block editor on a different page, e.g. the frontend?

Maybe we should check if Gutenberg is enqueued on the page, and enqueue it in that case.